addons/tmf_loadouts/loadouts/ru_army_template_2020.hpp

```diff
@@ -555,13 +555,15 @@ class sp : g {
 	};
 };
 
-class vg : smg {
+class vg : car {
 	displayName = "Vehicle Gunner";
 	goggles[] = {"default"};
-	vest[] = {"CUP_V_RUS_6B45_1"};
-	headgear[] = {"H_Tank_black_F"};
+	vest[] = {"CFP_V_O_RUMVD_SURPAT"};
```

Comment:
I'd say that the interceptor vest looks more like the 6b48 than this

Reply:
Hmmm... good point. Maybe replace it with "CUP_V_B_Interceptor_Base_Olive"?

Reply:
Maybe, check for clipping first.

Reply:
Yeah, looks like the CUP PASGT is better quality/less clipping. Swapping to that instead.